docs/user/how-to/run-container.rst

```diff
@@ -3,13 +3,13 @@ Run in a container
 
 Pre-built containers with PandABlocks-ioc and its dependencies already
 installed are available on `Github Container Registry
-<https://ghcr.io/PandABlocks/PandABlocks-ioc>`_.
+<https://ghcr.io/pandablocks/pandablocks-ioc>`_.
 
 Starting the container
 ----------------------
 
 To pull the container from github container registry and run::
 
-    $ docker run ghcr.io/PandABlocks/PandABlocks-ioc:main --version
+    $ docker run https://ghcr.io/pandablocks/pandablocks-ioc:main --version
```

Comment:
This doesn't seem to work - this format is unrecognised, for `podman` at the very least.

Reply:
podman doesn't take https://, fixed in the recent squash